lib/registries/addon/components/editable-field/template.hbs

```diff
@@ -0,0 +1,20 @@
+{{#if @registration.userHasAdminPermission}}
+    <div local-class='Editable'>
+        <h4>{{t @title}}</h4>
+        {{#if @registration.userHasAdminPermission}}
```

Comment:
This is checked twice -- could you remove the outer one, so there's only one `h4` and one `yield`?